tests/includes/wait-for.sh

```diff
@@ -12,10 +12,13 @@ wait_for() {
     name=${1}
     query=${2}
 
-    # shellcheck disable=SC2143
+    attempt=0
+    # shellcheck disable=SC2046,SC2143
     until [ "$(juju status --format=json 2> /dev/null | jq "${query}" | grep "${name}")" ]; do
-        juju status --relations
+        echo "[+] (attempt ${attempt}) polling status"
```

Comment:
Great, this did concern me, as we could just continue doing this for ever. With this change a future change could add a max attempt.